xep-0060.xml

```diff
@@ -2387,6 +2395,17 @@ And by opposing end them?
     </items>
   </pubsub>
 </iq>
+]]></example>
+    <p>If there are fewer items in the node than requested, the service should just return all of them.</p>
+    <example caption='Service returns no items if it has none'><![CDATA[
```

Comment:
Nit: "Service has no items" or similar would be shorter, it obviously can't return any items if it doesn't have any.